interface/wx/cursor.h

```diff
@@ -138,11 +138,8 @@ class wxCursor : public wxGDIObject
               (to load a cursor from a .ico icon file) and @c wxBITMAP_TYPE_ANI
               (to load a cursor from a .ani icon file).
             - under MacOS, it defaults to @c wxBITMAP_TYPE_MACCURSOR_RESOURCE;
-              when specifying a string resource name, first the color cursors 'crsr'
-              and then the black/white cursors 'CURS' in the resource chain are scanned
-              through. Note that resource forks are deprecated on macOS so this
-              is only available for legacy reasons and should not be used in
-              new code.
+              when specifying a string resource name, first 'png' and then 'cur'
+              image are searched in resources.
```

Comment:
I'm not an authority on wording, it's a foreign language to me as well, but I think either we go 
.. first a PNG and then a CUR image is searched or
... first PNG and then CUR images are searched
I'm slightly preferring the first variant

Reply:
@csomor ,
I followed you first suggestion and fixed it.

Hopefully someone will be able to push this before 3.1.7...

Thx.